gp-nested-forms/gpnf-display-child-entries-table-format-all-child-fields.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Gravity Perks // GP Nested Forms // Display Table Format for All Fields
+ * https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ *
+ * MODIFIED: Show *all* child fields.
```

Comment:
We should include the full description from the [original snippet](https://github.com/gravitywiz/snippet-library/pull/61/files) and modify it as necessary for this version.

Reply:
I have added the description but I had to modify it. This snippet actually works without the gpnf_table modifier, so I removed that part from the description. Please, is this okay?